odoo/addons/base/ir/ir_qweb/qweb.py

```diff
@@ -99,20 +99,35 @@ def visit_Lambda(self, node):
         if args.vararg: names.append(args.vararg)
         if args.kwarg: names.append(args.kwarg)
         # remap defaults in case there's any
-        return ast.copy_location(ast.Lambda(
-            args=arguments(
-                args=args.args,
-                defaults=[self.visit(default) for default in args.defaults],
-                vararg=args.vararg,
-                kwarg=args.kwarg,
-                # assume we don't have any, not sure it's even possible to
-                # handle that cross-version
-                kwonlyargs=[],
-                kw_defaults=[],
-                posonlyargs=[],
```

Comment:
Fix the `arguments` shim instead (higher in the file). Either add `posonlyarg`, or replace `kwonlyargs, kw_defaults` by an ignored `**kwargs` at the end of the call.

Reply:
Also you'll probably want to fw-bot ignore as the file was moved in 12.0 (and the shim was apparently removed in saas-12.1 / 13.0: https://github.com/odoo/odoo/commit/52f5528cfb6d1152ee226e268247dc685a8ba329#diff-56c5a32de746052fcf317821d868e243L38-L49)